smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/SmithyJSONTypes.kt

```diff
@@ -9,13 +9,18 @@ import software.amazon.smithy.swift.codegen.SwiftDeclaration
 import software.amazon.smithy.swift.codegen.SwiftDependency
 
 object SmithyJSONTypes {
-    val Writer = runtimeSymbol("Writer", SwiftDeclaration.CLASS)
-    val Reader = runtimeSymbol("Reader", SwiftDeclaration.CLASS)
+    val Writer = runtimeSymbol("Writer", SwiftDeclaration.CLASS, listOf(SmithyReadWriteTypes.SmithyWriter))
+    val Reader = runtimeSymbol("Reader", SwiftDeclaration.CLASS, listOf(SmithyReadWriteTypes.SmithyReader))
```

Comment:
Smithy JSON Reader & Writer are made SPI and import SmithyReader / SmithyWriter along with themselves.